tests/suite/test_suite.py

```diff
@@ -8,20 +8,16 @@
 
 from inspect import signature, Parameter
 
-# TODO: For now, these are hardcoded. Indeed, late join after recovery is not yet supported.
-# https://github.com/microsoft/CCF/issues/315
+# TODO: For now, these are hardcoded.
+# https://github.com/microsoft/CCF/issues/712
```

Comment:
This hardcoded list is now missing most of the tests (`e2e_logging`, `add_node_untrusted_code`, `retire_node`). I think the hardcoded version should be doing 'everything' at least once?

Reply:
The test cases I have deleted are already tested as part of the `code_update_test` and `reconfiguration` end-to-end tests so I don't think they are needed here. Because the cost of setting up a network is fairly expensive, we might want to skip these standalone end-to-end tests at some point and only use the suite though.

For the `e2e_logging` test, my plan is to build a new transactions object that records all the transactions issued to the network since the beginning of the suite and can check whenever required (i.e. after recovery) that the transactions are still in CCF.